oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBStorageConfig.java

```diff
@@ -47,5 +47,17 @@ public class BanyanDBStorageConfig extends ModuleConfig {
      * Max size of {@link org.apache.skywalking.oap.server.core.query.type.ProfileTask} to be fetched
      * in a single request.
      */
-    private int profileTaskQueryMaxSize = 200;
+    private int profileTaskQueryMaxSize;
+    /**
+     * Shards Factor for measure/metrics.
+     */
+    private int metricsShardsFactor;
+    /**
+     * Shards Factor for a normal record.
+     */
+    private int recordShardsFactor;
+    /**
+     * Shards Factor for a super dataset
+     */
+    private int superDatasetShardsFactor;
```

Comment:
<img width="662" alt="image" src="https://user-images.githubusercontent.com/5441976/169683842-caf90d51-ac6e-406c-930b-4f6408016ef1.png">

Please take another look at how `factor` works in ElasticSearch. The factor is the amplification coefficient. Others are using `ShardsNumber`.

Reply:
@lujiajing1126 Your new commit seems missing this.

Reply:
Fixed